### PR TITLE
Add support for AWS Elasticsearch

### PR DESCRIPTION
**Pull request by bloodcarter**
Hi! This is my first pull request. Not sure if I am doing right.

### CODE REVIEW

**Comment by bloodcarter** [comment]
Done. Check it out, please.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino-elasticsearch",
-  "version": "2.1.0",
+  "version": "2.2.6",
```
Comment (mcollina) [inline package.json]: can you please restore this as well?

File: pino-elasticsearch.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const split = require('split2')
 const pump = require('pump')
 const fs = require('fs')
 const path = require('path')
+let AWS = require('aws-sdk');
```
Comment (mcollina) [inline pino-elasticsearch.js]: this should be `const`.

File: package.json [hunk 3]
```diff
@@ -34,7 +34,9 @@
     "minimist": "^1.2.0",
     "pump": "^1.0.2",
     "readable-stream": "^2.3.3",
-    "split2": "^2.1.1"
+    "split2": "^2.1.1",
+    "aws-sdk": "^2.83.0",
+    "http-aws-es":"^3.1.0"
```
Comment (mcollina) [inline package.json]: is this dependency 'http-aws-es' needed?

File: README.md [hunk 4]
```diff
@@ -27,6 +27,7 @@ npm install pino-elasticsearch -g
   -t  | --type              the name of the type to use; default: log
   -b  | --size              the number of documents for each bulk insert
   -l  | --trace-level       trace level for the elasticsearch client, default 'error' (info, debug, trace).
+  -c  | --aws-credentials   path to aws_config.json (is using AWS Elasticsearch)
```
Comment (mcollina) [inline README.md]: can you add a quick example on how to use this with AWS?

it should be "if using AWS Elasticsearch".

**Comment by bloodcarter** [comment]
Done. Yes, "http-aws-es" is a dep.

**Comment by mcollina** [comment]
Thanks!!!!

**Comment by bloodcarter** [comment]
Wow. Cool :) So fast :)
Will it be published automatically?

**Comment by mcollina** [comment]
v2.2.0 - published right now.

**Comment by bloodcarter** [comment]
Thank you for beling so fast! You saved my day! BTW, I think it makes sense to update https://github.com/pinojs/pino/blob/master/docs/transports.md. Should I do a PR too?

**Comment by mcollina** [comment]
sure, go ahead.

**Pull request merged**